python/graphstorm/eval/evaluator.py

```diff
@@ -604,7 +604,7 @@ def setup_task_tracker(self, client):
         self.tracker = client
 
     @abc.abstractmethod
-    def evaluate(self, val_scores, test_scores, total_iters):
```

Comment:
You should not change val_scores and test_scores to others. Each Evaluator has evaluate func and has the same set of arguments. This is a an abstract function.

Reply:
Reverted.